src/test/java/com/amihaiemil/zold/mock/Response.java

```diff
@@ -97,64 +93,58 @@ public Response(final int status, final String jsonPayload) {
     }
 
     @Override
-    public StatusLine getStatusLine() {
```

Comment:
@wholegroup , it looks like all `statusLine`, `statusCode`, `entity`, `getProtocolVersion`, `params` methods have been removed completely. At the same time, new ones have been introduced: `getCode`, `setCode`, `getVersion` (a renamed `getProtocolVersion`?) and, maybe, some others which I missed. What was the reason of that refactoring in the scope of this issue?

Reply:
API was changed and there are not such methods anymore, that's why I removed them and used others.